config/routes.d/staging.horse.json

```diff
@@ -5,7 +5,8 @@
       "^/docs/private-cloud/": "developer.rackspace.com/user-guide.html",
       "^/docs/api-doc-template/": "developer.rackspace.com/user-guide.html",
       "^/docs/billing-api/": "developer.rackspace.com/user-guide.html",
-      "^/docs/sdp/": "developer.rackspace.com/user-guide.html"
+      "^/docs/sdp/": "developer.rackspace.com/user-guide.html",
+      "^/docs/rpc-vmware/server-virt-handbook/": "developer.rackspace.com/user-guide.html"
```

Comment:
Sorry! One more:
s/`^/docs/rpc-vmware/server-virt-handbook`/`^/docs/managed-vmware-services/server-virt-handbook/`
The key here in the routes file should match the key in the content file above.